phpstan.neon

```diff
@@ -46,7 +46,9 @@ parameters:
       message: 'Use PSR-7 API instead'
   disallowedStaticCalls:
     -
-      method: 'TYPO3\CMS\Extbase\Utility\DebuggerUtility::var_dump()'
+      method:
+        - 'TYPO3\CMS\Extbase\Utility\DebuggerUtility::var_dump()'
+        - '\TYPO3\CMS\Core\Utility\DebugUtility::debug()'
```

Comment:
This should be consistent - either with or without a leading backslash.

Reply:
Absolutely. I have added the backslash to the DebuggerUtility, because it is the FQDN.

Reply:
I've just had a look into the documentation of the PHPStan extension: There, they use it without the trailing backslash. I'd propose we stick to that.

https://github.com/spaze/phpstan-disallowed-calls/blob/main/docs/custom-rules.md

(I've also just tested that this works.)

Reply:
Right, done.